src/components/molecules/Settings/Workspace/AssetSection/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import AssetContainer from "@reearth/components/molecules/Common/AssetModal/AssetContainer";
+
+type Asset = {
+  id: string;
+  teamId: string;
+  name: string;
+  size: number;
+  url: string;
+  contentType: string;
+};
+
+type Props = {
+  assets?: Asset[];
+  onCreate?: (files: FileList) => void;
+  onRemove?: (assets: Asset[]) => void;
```

Comment:
```suggestion
  onRemove?: (assets: string[]) => void;
```

I think the ID is enough to identify the target of deletion.